manifests/example-dev/example-modules.yaml

```diff
@@ -27,7 +27,7 @@ parameters:
         key: FullAccessPolicyArn
 ---
 name: docker-build
-path: modules/examples/docker-build/
+path: git::https://github.com/awslabs/idf-modules.git//modules/containers/ecr?ref=release/1.3.0&depth=1
```

Comment:
We _will_ need to cut 1.3.0 before merge

Reply:
Thanks, will do!